src/pilot/assign_role.py

```diff
@@ -267,15 +267,24 @@ def define_target_raid_config(role, drac_client):
         'logical_disks': logical_disks} if logical_disks is not None else None
 
 
-def is_raid_controller(fqdd):
-    return fqdd.startswith('RAID.')
+def is_raid_controller(fqdd, drac_client):
+    if fqdd.startswith('RAID.'):
+        return True
+    elif fqdd.startswith('BOSS'):
+        return True
+    else:
+	disk_controllers = drac_client.list_raid_controllers()
+        boss_raid_controller = [
+            c.id for c in disk_controllers if c.model.startswith('BOSS')]
+        if fqdd == boss_raid_controller[0]:
+            return True
```

Comment:
Is this else clause needed?  Isn't the check to see if the FQDD starts with BOSS above enough?

Reply:
ok. got it.